musicbrainz_bot/editing.py

```diff
@@ -325,6 +325,30 @@ def edit_area(
         Returns:
             str: returns the area MBID of the edited area.
         """
+
+        # NOTE this method DOESN'T REMOVE OLD AREA URLS, as it requires a relationship_id.
+        # Which has to be queried through the database.
+        # I am ways to fix this, but currently new urls can only be added, and not removed.
```

Comment:
How is it going with this? :)

Reply:
I think it's fine to merge this PR if it's working for *adding* URLs, for now, but then I'd remove this WIP code and put it in another branch to propose later once it's working.

Reply:
It would help keep the git history clean. But we can do it before merging.